kolibri/plugins/facility/assets/src/composables/useUserManagement.js

```diff
@@ -37,18 +38,28 @@ export default function useUserManagement({
     dataLoading.value = true;
     try {
       const fetchResource = softDeletedUsers ? DeletedFacilityUserResource : FacilityUserResource;
+      const getParams = by_ids.value
+        ? {
+            by_ids: by_ids.value,
+            member_of: activeFacilityId,
+            page: page.value,
+            page_size: pageSize.value,
+          }
+        : pickBy({
+            member_of: activeFacilityId,
+            date_joined__gte: dateJoinedGt?.toISOString(),
+            page: page.value,
+            page_size: pageSize.value,
+            search: search.value?.trim() || null,
+            ordering: order.value === 'desc' ? `-${ordering.value}` : ordering.value || null,
+            ...getBackendFilters(),
+          });
```

Comment:
Just noting that since we are not sending the `ordering` param, the column sorting is not working properly in this view.


https://github.com/user-attachments/assets/db86885a-557e-4e7d-950f-eb2b836ac987

Reply:
Thank you!!